### PR TITLE
create TabPane in ViewLibrary only once

### PR DESCRIPTION
**Pull request by plata**
Fixes issue that tab of the installation wizard is closed before the installation is finished (see PlayOnLinux/Scripts#260).

### CODE REVIEW

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewLibrary.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 
     private LibrarySideBar sideBar;
     private LibraryPanel libraryPanel;
+    private final Tab installedApplication;
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewLibrary.java]: `installedApplicationsTab` right?

Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewLibrary.java]: yes

**Comment by plata** [comment]
Maybe we could bind the content instead of setting it when something changes?

**Comment by madoar** [comment]
You mean instead of `this.installedApplicationsTab.setContent(availableShortcuts);`?
I'm not sure why this line is necessary currently, because we're already using a binding to set the content of `availableShortcuts`.

**Comment by plata** [comment]
I don't know. Maybe it isn't. Just fixed the error without really thinking about it.

**Comment by madoar** [comment]
Can you try to remove the row and see if the behavior is still the same?

**Comment by plata** [comment]
It's not working (nothing shown at all).

**Comment by madoar** [comment]
Hm strange.
Maybe we should open an issue for this (showing the `availableShortcuts` through the use of a binding).

**Pull request merged**